### ISSUE TITLE
Comments on section 3.3 “Accessibility Requirements”

### ISSUE THREAD

**Issue by rdeltour** [w3c/wcag-act#135]
- The following statement is a bit awkward IMO:
> An ACT Rule MAY be a complete or partial test for one or more accessibility requirements.

An ACT Rule **is always** a complete or partial test. What about instead:

> An ACT Rule MAY test only part of an accessibility requirement, or MAY cover more than one requirement.

- I'm not sure we can use a normative MUST on the issue of a rule in the following statement:
> e.g. if the rule returns a Failed result, this MUST be a failure for the accessibility requirement.

I would suggest instead:
> In other words, evaluating the rule will result in a Failed outcome only if the evaluated content fails the accessibility requirement.

- Failing to parse:
> in these too would be accessibility requirements.

**Comment by WilcoFiers** [comment]
Proposed change: "An ACT Rule is a complete or partial test for one or more accessibility requirements."

**Comment by nitedog** [comment]
For bullet 2: "a rule only returns the outcome Fail when the content fails the accessibility requirement"
For bullet 3: "Some organizations have additional Accessibility Requirements, such as specific implementation techniques to meet WCAG 2 Success Criteria."

**Comment by maryjom** [comment]
@moekraft We have agreed on Wilco's and Shadi's proposed edits above. Please make these changes to the spec.

**Comment by moekraft** [comment]
Addressed by PR #146

**Comment by moekraft** [comment]
Addressed by PR #146
